README.md

```diff
@@ -47,6 +47,16 @@ stack_snapshot(
     snapshot = "lts-18.28",
 )
 ```
+Right now there are also some issues with `gazelle` and as a temporary fix you need to include the following in your `WORKSPACE`:
+```python
+go_repository(
+    name = "org_golang_x_xerrors",
+    importpath = "golang.org/x/xerrors",
+    sum = "h1:go1bK/D/BFZV2I8cIQd1NKEZ+0owSTG1fDTci4IqFcE=",
+    version = "v0.0.0-20200804184101-5ec99f83aff1",
+)
+```
```

Comment:
Should this appear before calling `gazelle_dependencies()`?

Reply:
Seems to work both before and after `gazelle_dependencies()`. Additionally, would it be possible to somehow temporarily include this in `gazelle_cabal_dependencies`? :thinking:

Reply:
It could interfere if the user wanted to use a different version of `golang_x_xerrors`, I guess.

Reply:
I guess we can just merge this as-is then?